exercises/03.async/02.solution.rejections/README.mdx

```diff
@@ -1,14 +1,32 @@
-# Promise rejections
+# Handling Promise Rejections
+
+<EpicVideo url="https://www.epicweb.dev/workshops/testing-fundamentals/testing-asynchronous-code/error-assertions-for-rejected-promises/solution" />
 
 The first thing to realize is that with the `.rejects.toThrow()` API we are creating _a chain_. The `rejects` part becomes a property that returns another object with the `.toThrow()` function. I reflect this in the `Assertions` type straight away:
 
-<CodeFile file="setup.ts" range="1-6" highlight="3-5" />
+```typescript
+interface Assertions {
+	toBe(expected: unknown): void
+	rejects: {
+		toThrow(expected: Error): Promise<void>
+	}
+}
+```
 
 > Note that the `.rejects.toThrow()` assertion will be returning a Promise. Annotate its return type accordingly (`Promise<void>`).
 
 Next, I extend the object returned from the `globalThis.expect` function to have the `rejects` property. In that property, I define a new `toThrow` function.
 
-<CodeFile file="setup.ts" range="15-23" highlight="22-23" />
```

Comment:
Shoot, was that pointing at something broken before?

Reply:
I rarely use the `CodeFile` component because I feel it's took complex. I'd prefer to just put the code in directly like @Creeland is doing.

Reply:
Should we then do a separate PR to migrate from `Codefile` consistently throughout the entire workshop? I can do that.

Reply:
Thank you!